client/app/controllers/collection/add.js

```diff
@@ -4,8 +4,7 @@ export default Ember.Controller.extend({
     addMethod: 'select', // 'select' or 'create'
     methodSelected: false,
     type: Ember.computed('model.settings', function(){
-        let type = this.get('model.settings.collectionType').toLowerCase();
-        return  type === 'preprint' || type === 'project' ? 'node' : type;
+        return this.get('model.settings.collectionType').toLowerCase();
```

Comment:
Can you handle the case where collectionType is not set? You can add a default so the application doesn't break. There would be a more robust way to handle that in the full application.